### ISSUE TITLE
[TB] Molecules - Slider - Missing labels, causing accessibility issues

### ISSUE THREAD

**Issue by kburk1997** [finos/a11y-theme-builder#955]
## Problem/Concern
Both the single and range sliders are unlabeled, resulting in automated checker errors:
### Issue
`Ensures every form element has a label` ([label](https://accessibilityinsights.io/info-examples/web/label))
### Target application
[A11y ThemeBuilder](http://localhost:3000/designSystem/My%20Design)
### Element path
input[value="30"]
### Snippet
`<input data-index="0" aria-valuenow="30" aria-orientation="horizontal" aria-valuemax="100" aria-valuemin="0" type="range" min="0" max="100" step="1" value="30" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 100%; margin: -1px; overflow: hid...`
### How to fix
    Fix any of the following:
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit label

### Screenshot
![Screenshot of unlabeled sliders on Slider page](https://github.com/finos/a11y-theme-builder/assets/11298269/acdf7160-5024-4924-9169-1bf22b9ceaf7)

## Proposed Solution
For single slider - have slider input use aria-labelledby="id of slider text""

For range slider - use aria-label="Start \<label\>" and "End \<label\>"
